src/line_break.c

```diff
@@ -0,0 +1,279 @@
+#include "global.h"
+#include "line_break.h"
+#include "text.h"
+#include "malloc.h"
+
+void StripLineBreaks(u8 *src)
+{
+    u32 currIndex = 0;
+    while (src[currIndex] != EOS)
+    {
+        if (src[currIndex] == 0xFA || src[currIndex] == 0xFE)
+            src[currIndex] = 0;
```

Comment:
Let's avoid using magic numbers for the chars.
```suggestion
        if (src[currIndex] == CHAR_PROMPT_SCROLL || src[currIndex] == CHAR_NEWLINE)
            src[currIndex] = CHAR_SPACE;
```